src/productcard.css

```diff
@@ -0,0 +1,44 @@
+@import url("https://fonts.googleapis.com/css2?family=Karla&family=Roboto+Mono:wght@300&family=Tajawal&display=swap");
```

Comment:
You can put "global" CSS in the `index.css` file - then it will be available in all components so you don't need to import fonts in each stylesheet :)